guides/02_building-interfaces/02_reactive-interfaces.md

```diff
@@ -19,4 +19,6 @@ The difference between `gr.Audio(source='microphone')` and `gr.Audio(source='mic
 
 Here is example code of streaming images from the webcam.
 
-$code_stream_frames
\ No newline at end of file
+$code_stream_frames
+
+Streaming can also be done in an output component. A `gr.Audio(streaming=True)` output component can take a stream of audio data yielded piece-wise by a generator function and combines them into a single audio file.
```

Comment:
Let's put the `stream_audio_out` example demo here (ideally after simplifying it a bit)